packages/common/src/ui/layout/tables/columns/NotePopoverColumn/NotePopoverColumn.tsx

```diff
@@ -47,13 +65,19 @@ export const getNotePopoverColumn = <T extends DomainObject>(
     const t = useTranslation('common');
     const value = column.accessor({ rowData, rows });
 
+    let content: NoteObject[] = value as NoteObject[];
+    if (!Array.isArray(value)) {
+      content = [value as NoteObject];
+    }
+
     return value ? (
       <PaperPopover
         width={400}
-        height={180}
         Content={
           <PaperPopoverSection label={label ?? t('label.notes')}>
-            {String(value)}
+            {content.map(({ header, body }) => (
```

Comment:
could you filter out blank bodies please? otherwise you can get something like this:

<img width="542" alt="Screen Shot 2021-12-23 at 4 03 31 PM" src="https://user-images.githubusercontent.com/9192912/147182900-f3d72989-5a03-43e7-b240-7d929c476d84.png">

Reply:
Ah, thanks for noticing this.

I've filtered out the sections with no note, but left the ones which don't have a batch. Not ideal, but not kind of need to differentiate and if theres no batch code, not sure what to use 🙄